### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.4.1.2 - add aria-current to carousel

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1315]
# Feature Request

##ToDo:
Active slides should have aria-current="true".

**Comment by SarahEdding** [comment]
QA done, all fixed
